### ISSUE TITLE
Disabled condition does not include aria-disabled

### ISSUE THREAD

**Issue by etienne-dldc** [chakra-ui/panda#3270]
### Description

Hi,

The [doc explicitly mentioned](https://panda-css.com/docs/concepts/conditional-styles#aria-attribute) `aria-disabled` to be handled by the `_disabled` condition:

> [!TIP]
> Most of the `aria-{state}` attributes typically mirror the support ARIA states in the browser pseudo class. For example, `aria-checked=true` is styled with `_checked`, `aria-disabled=true` is styled with `_disabled`. 

Yet [the table that follow](https://panda-css.com/docs/concepts/conditional-styles#reference) does not:

| Condition name         | Selector                                                           |
| ---------------------- | ------------------------------------------------------------------ |
| \_disabled             | `&:is(:disabled, [disabled], [data-disabled])`                     |

And in fact, the default `_disabled` does not include `aria-disabled` (ie. the table is right).

This looks like:
- Either a bug and `aria-disabled` should be included (in `_groupDisabled` and `_peerDisabled` as well I guess).
- Or this is on purpose in which case the doc should be corrected (and I'm curious to know the reasoning behind...)

Happy to open a PR if I get a clear response on which of the 2 direction is the right one.

## Workaround

The easy workaround is to re-declare the `_disabled` condition like this:

```ts
export default defineConfig({
  conditions: {
    extend: {
      disabled: "&:is(:disabled, [disabled], [data-disabled], [aria-disabled=true])",
    },
  },
});
```

### Link to Reproduction

https://play.panda-css.com/KWKUDpA3cK

### Steps to reproduce

1. Visit example above
2. See that the button with `aria-disabled` does not get the red background applied.

### JS Framework

React (but apply to all)

### Panda CSS Version

0.53.7

### Operating System

- [x] macOS
- [ ] Windows
- [ ] Linux

**Comment by segunadebayo** [comment]
Good catch @etienne-dldc 

Love the see the migration PR in your project.

I've pushed a fix for this and will do a release shortly.
